scalableminds/webknossos
# Only apply blur on enter/esc behaviour to vector inputs and tree/group name input

This is a follow-up PR for [4141](https://github.com/scalableminds/webknossos/pull/4141). It disables "blur on esc/enter" behavior for unwanted input components. The components left with this behavior are input vectors and the name input for tree/ group names in the trees tab.

### URL of deployed dev instance (used for testing):
- [https://makevectorinputsblurable.webknossos.xyz](https://makevectorinputsblurable.webknossos.xyz)

### Steps to test:
- Open a tracing.
- Try to edit and add linebreaks to the description via pressing enter. this should work out.
- Edit the position and rotation vector input in e.g. flight mode. Try to blur it with esc and enter. This should also work out. Also try this behavior for the name input of trees/groups in the trees tab.

------
